util/notify/notify.lisp

```diff
@@ -96,6 +96,19 @@
   (:name "GetCapabilities")
   (values #("actions" "action-icons" "body" "body-markup")))
 
+;;; based on dbus notifications spec
+;;; https://specifications.freedesktop.org/notification-spec/latest/ar01s09.html
```

Comment:
I'd rather we link to the spec in the README

Reply:
Thanks, I can add the spec link to the README as well, as a separate PR perhaps.